src/planning/nav2_dijkstra_planner/src/Logger.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef LOGGER_H_
+#define LOGGER_H_
+
+#include <cstdio>
+
+// TODO(orduno): Figure out how to get a reference to the node logger
```

Comment:
We shouldn't need this logger.h file at all. If you include rclcpp.hpp then you can call rclcpp::get_logger() to get the logger. I filed issue #60 to track this.